plugins/common/common.go

```diff
@@ -25,11 +25,16 @@ type ShellCmd struct {
 // NewShellCmd returns a new ShellCmd struct
 func NewShellCmd(command string) *ShellCmd {
 	items := strings.Split(command, " ")
-	cmd := items[0]
-	args := items[1:]
+	return NewTokenizedShellCmd(items...)
+}
+
+// NewTokenizedShellCmd creates a new ShellCmd
+func NewTokenizedShellCmd(command ...string) *ShellCmd {
```

Comment:
if `NewShellCmd` just calls `NewTokenizedShellCmd` why not just update `NewShellCmd`

Reply:
I'm gonna just get rid of this. It was vestigial from when I was making some calls to `dokku` rather than `plugn`. I might open something similar as another PR in the future.